src/docker/processor/Dockerfile

```diff
@@ -0,0 +1,55 @@
+### Indexer Processor Image ###
```

Comment:
This was copied (and partly edited for directories) from the dockerfile given to us by the processor project as it is.